mica-webapp/src/main/webapp/index.html

```diff
@@ -212,8 +212,9 @@
 <script src="bower_components/angular-xeditable/dist/js/xeditable.js"></script>
 <script src="bower_components/angular-media-queries/match-media.js"></script>
 <script src="bower_components/sf-localized-string/dist/sf-localized-string.min.js"></script>
-<script src="bower_components/sf-obiba-file-upload/dist/sf-obiba-file-upload.min.js"></script>
+<script src="bower_components/sf-obiba-file-upload/dist/sf-obiba-file-upload.js"></script>
```

Comment:
This is to be fixed in the add-on project, I guess it was changed for debugging purposes and left in the bower/package JSON. In reality all these scripts will be minimised in Mica2 project but .min will be best for other projects.